queue-manager/core/src/manager.ts

```diff
@@ -460,6 +476,11 @@ class Manager {
     return this.queues;
   }
 
+  public async deleteQueue(queue_id: QueueID) {
```

Comment:
```suggestion
  public deleteQueue(queue_id: QueueID) {
```
Please make a normal function